package.json

```diff
@@ -24,5 +25,10 @@
       "no-undef": 0,
       "no-unused-vars": 0
     }
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm test"
```

Comment:
The title and commit says pre commit, but this is s pre push.

Reply:
My mistake. Do you want me to update the commit message?